tests/Unit/Utilities/Test_FileSystem.cpp

```diff
@@ -0,0 +1,97 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+#include <fstream>
+
+#include "Utilities/FileSystem.hpp"
+#include "Utilities/Literals.hpp"
+
+TEST_CASE("Unit.Utilities.FileSystem.get_path", "[Unit][Utilities]") {
+  CHECK("/test/path/to/dir/"s ==
+        filesystem::get_path("/test/path/to/dir/dummy.txt"));
+  CHECK("/test/path/to/dir/"s == filesystem::get_path("/test/path/to/dir/"));
```

Comment:
Yeah, it looks like I forgot to make a note on it, but even after looking at the source it was still not clear to me what `get_path` does or is supposed to do.